### PR TITLE
add some basic tests

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/encoding-down/issues/2

@juliangruber These tests are very rudimentary and we should probably add more. If you have any suggestions, create an issue and I'll fix.

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Tests fail on node 0.12 of course because no `Buffer.from()`

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    var db2 = encdown(memdb, { valueEncoding: 'json' })
+    db2.get('foo', function (err, value) {
+      t.equals('EncodingError', err.name)
+      memdb.close(t.end.bind(t))
+    })
+  })
+})
+
+test('can decode from string to json', function (t) {
+  var memdb = memdown()
```
Comment (ralphtheninja) [inline test/index.js]: I just realized that you can create multiple `encoding-down` and reuse the same backend. Cool! (So it's like a view?)

Comment (ralphtheninja) [inline test/index.js]: Reminds me of https://github.com/mafintosh/levelup-defaults a bit

**Comment by ralphtheninja** [comment]
@juliangruber Sorry to bother you, but would be nice to be owner on this one as well ;)

**Comment by juliangruber** [comment]
you're not bothering :) added you!

**Pull request merged**